sdk/trace/src/main/java/io/opentelemetry/sdk/trace/SdkTracerProvider.java

```diff
@@ -69,11 +72,28 @@ public Tracer get(String instrumentationName, @Nullable String instrumentationVe
     return tracerSdkComponentRegistry.get(instrumentationName, instrumentationVersion);
   }
 
+  /** Returns the active {@link TraceConfig}. */
   @Override
   public TraceConfig getActiveTraceConfig() {
     return sharedState.getActiveTraceConfig();
   }
 
+  /**
+   * Attempts to stop all the activity for this {@link Tracer}. Calls {@link
+   * SpanProcessor#shutdown()} for all registered {@link SpanProcessor}s.
+   *
+   * <p>This operation may block until all the Spans are processed. Must be called before turning
```

Comment:
It may block AND return `CompletableResultCode`?

Reply:
Nice catch